### PR TITLE
[PW-5347] update tooltips for new customer area UI

### PR DESCRIPTION
**Pull request by peterojo**
**Description**

Update tooltips for the paths that have changed in new CA interface
**Tested scenarios**

**Fixed issue**:

### CODE REVIEW

**Review by AlexandrosMor — APPROVED** [review]

File: etc/adminhtml/system/adyen_split_payment.xml [hunk 1]
```diff
@@ -29,7 +29,7 @@
         <comment>
             <![CDATA[
                 <p>
-                It is possible to do split payments with GiftCards please configure here what refund strategie you want to use
+                It is possible to do split payments with GiftCards please configure here what refund strategy you want to use.
```
Comment (acampos1916) [inline etc/adminhtml/system/adyen_split_payment.xml:32]: Rephrasing it a bit. And Split Payment is now related to AFP, the right label would be Partial Payment
```suggestion
                It is possible to do partial payments with GiftCards. Set here which refund strategy to use.
```

Comment (Aleffio) [reply etc/adminhtml/system/adyen_split_payment.xml]: Should the Label on line 27 and/or the filename be renamed as well?

**Review by AlexandrosMor — APPROVED** [review]

File: etc/adminhtml/system/adyen_split_payment.xml (hunk 1 above)
Comment (acampos1916) [reply etc/adminhtml/system/adyen_split_payment.xml]: Makes sense yes. Are you able to change the label and filename @peterojo ?

Comment (peterojo) [reply etc/adminhtml/system/adyen_split_payment.xml]: I can but there's quite a lot of references to split payments in the codebase, and we would also need to test that it still works.
Screen Shot 2021-09-09 at 3 46 37 PM

I would rather make it a separate effort. Here we just wanted to correct a typo 😄

Comment (peterojo) [reply etc/adminhtml/system/adyen_split_payment.xml]: I've created a ticket [PW-5431] to update all references to split payments

**Review by acampos1916 — APPROVED** [review]

**Pull request merged**